res/build.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 # uses PREFIX from environment
```

Comment:
Update the comment.

Reply:
to what?

Reply:
`uses PREFIX and BUILD_MANPAGES from environment`

Reply:
done